modules/identity-provider-create-CRD.adoc

```diff
@@ -24,18 +24,30 @@ Resource Definition (CRD) that describes it.
 
 .Procedure
 
-. Log in to the cluster as:
-+
-[source,bash]
-----
-$ command
-----
+Create a CRD file to describe the identity provider. A generic file displaying
+the structure is below.
 
-. Create the CRD for your identity provider:
-+
-[source,bash]
 ----
-$ command
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  identityProviders:
+  - name: my_identity_provider <1>
+    challenge: true <2>
+    login: true <3>
+    mappingMethod: claim <4>
+    type: <type> <5>
+    ...
 ----
+<1> A unique name defining the identity provider. This provider name is
+prefixed to provider user names to form an identity name.
+<2> When `true`, unauthenticated token requests from non-web clients (like the
```

Comment:
no parentheses

Reply:
Incorporated. I've also removed parentheses from similar locations in the other CRD files.